app/src/main/java/ru/vssemikoz/newsfeed/models/NewsItemList.java

```diff
@@ -0,0 +1,15 @@
+package ru.vssemikoz.newsfeed.models;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class NewsItemList {
+    @SerializedName("articles")
+    private List<NewsItem> newsItem;
```

Comment:
It's better to create a set of other classes for parsing API responses as they are subject to change and your business logic and business models should not know about it and should not change because of it. So, you'll have two sets of models and API models should be converted into the business models in some method in case of successful network call.